source/dofs/dof_tools_constraints.cc

```diff
@@ -1083,6 +1085,77 @@ namespace DoFTools
         }
     }
 
+    namespace internal
+    {
+      /**
+       * Try to find a finite element inside FECollection of @p dof_handler
+       * which is dominating those in @fes .
```

Comment:
There may be multiple. Say you have a collection that contains `{Q1,Q2,Q3,Q4}` and you're asking for the dominating one for `{Q3,Q4}`. Any of `{Q1,Q2,Q3}` satisfies that role. You want the *least dominant* of these.